SimPEG/EM/Utils/AnalyticUtils.py

```diff
@@ -1,4 +1,5 @@
-from SimPEG import Mesh, np, Utils
+import numpy as np
+from SimPEG import Mesh, Utils
```

Comment:
I don't think it is essential to do for this pr; we can work on getting it in later on